src/main/java/org/isf/telemetry/gui/TelemetryGUI.java

```diff
@@ -84,17 +85,13 @@ public class TelemetryGUI extends JDialog {
 	private JPanel panel;
 	private TelemetryManager telemetryManager = Context.getApplicationContext().getBean(TelemetryManager.class);
 	private TelemetryUtils telemetryUtils = Context.getApplicationContext().getBean(TelemetryUtils.class);
-
-	public TelemetryGUI() {
-		super();
-		init();
-	}
```

Comment:
I think this constructor is needed to open the window from Main Menu > Settings, could you please verify?